terraform/modules/rds/main.tf

```
@@ -0,0 +1,56 @@
+# Non-compliant RDS configuration (missing encryption, backups, and monitoring)
+resource "aws_db_instance" "main" {
+  identifier        = "${var.environment}-db"
+  engine           = "postgres"
+  engine_version   = "13.7"
+  instance_class   = var.instance_class
+  allocated_storage = 20
+
+  # Non-compliant: Credentials in plain text
+  name     = var.db_name
+  username = "admin"  # Hardcoded credentials
+  password = "insecure123!"  # Hardcoded password
+
+  # Non-compliant: Public access enabled
+  publicly_accessible = true
+
+  # Non-compliant: No encryption
+  storage_encrypted = false
+
+  # Non-compliant: Minimal backup settings
+  backup_retention_period = 0
```

The RDS instance has a backup retention period of 0, which does not meet the minimum requirement of 30 days for compliance.

```suggestion
  backup_retention_period = 30
```